### PR TITLE
Add file repository in animal history reporting

### LINKED ISSUE #250: [NIRC] Add file repository
### Tasks
- [x] Update Animal History reporting to include file repository
- [x] Configure file repo (see JHU for example)
- [x] move file repository reporting to shared ehrModules
- [x] Code Review
- [x] Code Review changes
- [x] Test update?
- [x] TC verify and Merge

### PR DESCRIPTION
**Pull request by ankurjuneja**
#### Rationale
Add file repository in animal history reporting

### CODE REVIEW

File: nirc_ehr/resources/web/nirc_ehr/nircReports.js [hunk 1]
```diff
@@ -196,4 +196,173 @@ EHR.reports.clinicalHistory = function(panel, tab, showActionsBtn, includeAll){
...
+    }
+    else {
+        tab.add({
+            html:'No animal files found.',
+            border : false
+        });
+    }
+
+    const folders = [
+        "Misc Docs",
```
Comment (labkey-martyp) [inline nirc_ehr/resources/web/nirc_ehr/nircReports.js]: Can you also add Arrival Docs

**Review by labkey-martyp — COMMENTED** [review]
Sorry I might have directed you to the wrong place, this [version](https://github.com/LabKey/snprcEHRModules/blob/51f83c749332b081a7f1c913e82592fb19ce20b1/snprc_ehr/resources/web/snprc_ehr/snprcReports.js#L291) is a bit more efficient. Can you check it out please for this.

**Comment by labkey-bpatel** [comment]
The FileRepository function is identical to JHU's - does it make sense to push it to the EHR module? @ankurjuneja @labkey-martyp

**Comment by labkey-bpatel** [comment]
> The FileRepository function is identical to JHU's - does it make sense to push it to the EHR module? @ankurjuneja @labkey-martyp

Didn't notice Marty's comment about using SNPRC's version. In any case, my question still holds.

**Comment by labkey-martyp** [comment]
> The FileRepository function is identical to JHU's - does it make sense to push it to the EHR module? @ankurjuneja @labkey-martyp

I think that makes sense. The default folders need to be configurable but the rest is pretty much the same.

**Review by labkey-martyp — APPROVED** [review]

**Pull request merged**